environment/cloudformation/training/IndyCluster.json

```diff
@@ -1425,10 +1425,14 @@
                 "\"\n",
                 "apt-get update\n",
                 "DEBIAN_FRONTEND=noninteractive apt-get install -y debsigs debsig-verify apt-transport-https\n",
-                "DEBIAN_FRONTEND=noninteractive apt-get install -y dialog figlet python-pip python3-pip python3.5-dev libsodium18 unzip make screen indy-node tmux vim wget\n",
+                "DEBIAN_FRONTEND=noninteractive apt-get install -y dialog figlet python-pip python3-pip python3.5-dev unzip make screen sovrin tmux vim wget\n",
```

Comment:
It must install `indy-node`, not `sovrin`. `indy-node` contains all necessary dependencies and scripts.